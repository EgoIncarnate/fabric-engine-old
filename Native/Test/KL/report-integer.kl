function entry()
{
  report 3;
}
