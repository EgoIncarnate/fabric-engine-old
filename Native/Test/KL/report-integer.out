GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          ConstSize( 3 )
3
